project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/my_groups/my_groups.inc

```diff
@@ -21,7 +21,7 @@ $plugin = array(
 function c4m_features_homepage_my_groups_content_type_render($subtype, $conf, $args, $context) {
   $groups = c4m_features_homepage_get_user_groups();
 
-  if (is_null($groups)) {
+  if (empty($groups)) {
```

Comment:
Yes, that would also work, but the code would not be explicit. $groups is always an array, using !$groups could be understood as if $groups is a boolean.